ads/appnexus.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  writeScript, validateSrcPrefix, validateSrcContains
+}
+from '../src/3p';
+
+const APPNEXUS_AST_URL = 'https://acdn.adnxs.com/ast/ast.js';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function appnexus(global, data) {
+  // in case we pass the ttj url to use, simply call it and return
+  if (data.src) {
```

Comment:
This path is only expected for a single ad. If they are using multiple ads this path would not be used. Hard to estimate but less than 50% I would say.